### DISCUSSION TITLE
start from terminal

### DISCUSSION THREAD

**Discussion by DeuxMilles** [dail8859/NotepadNext#427] [General]
how can I start NotepadNext from the commandline?

**Comment by dail8859** [comment]
The most basic way is to use the full path to the EXE. Is this not working? What OS are you using, and how did you install it?

**Comment by DeuxMilles** [comment]
sorry, forgot to mention that: Linux Mint 21.2 Cinnamon.

**Reply by dail8859** [reply]
If you installed the flatpak, according to the [documentation](https://docs.flatpak.org/en/latest/using-flatpak.html) you can use:

> flatpak run com.github.dail8859.NotepadNext

If you have the AppImage, once you `chmod +x` the file to make it executable then it's just like any other exe.
